ui/components/AppBar.js

```diff
@@ -0,0 +1,24 @@
+import { AppBar } from '@mui/material';
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+import theme from '../styles/theme'
+import { appBarColor } from '../styles/colors';
+
+const CustomAppBarWrapper = styled(AppBar)(({theme})=>({
+    backgroundColor: appBarColor, 
+    position: "sticky",
+    elevation: 0,
+    padding: theme.spacing(1.4),
+    zIndex: 1100,
+}))
+
+export const CustomAppBar = () =>  { const theme = useTheme();
```

Comment:
In this case we need not do this. We can just export the CustomAppBarWrapper as it is. 
`export default CustomAppBarWrapper` and use it. Because, the `AppBar` component in mui requires `children` to be passed to it, but we are not doing it right now and it doesn't make sense for us to do it, because we are not making any changes other than some styling.

Reply:
Ok. I see, that's what you have done. My bad. then lets remove line 15 to line 21.

Reply:
Yeah, I was thinking the same we do not need this because in App bar and LoadingSpinner we just need to manipulate the styling a bit. Okay I am removing this.